docs/pages/reference/networking.mdx

```diff
@@ -36,21 +36,20 @@ following use cases:
 </TabItem>
 <TabItem scope={["cloud", "team"]} label="Cloud-Hosted">
 
-All Teleport services (e.g., the Application Service and Database Service) have
-an optional `public_addr` property that you can modify in each service's
-configuration file. The public address can take an IP or a DNS name. It can also
-be a list of values:
+For Teleport Enterprise (managed) the domain name and sub-domains
+are managed by Teleport for your account. The public address (`public_addr`)
+for the Teleport Application services are configurable.
```

Comment:
```suggestion
for the Teleport Application Service is configurable.
```
- Plurality agreement ("public address...is configurable")
- The convention is "Teleport Application Service"